### PR TITLE
doc: slm: fix XUDPCLI data reception format

### PR DESCRIPTION
**Pull request by ru-fu**
Fix a typo in the documentation for the XUDPCLI command.

Signed-off-by: Ruth Fuchss <ruth.fuchss@nordicsemi.no>

### CODE REVIEW

**Comment by ru-fu** [comment]
@junqingzou How about line 1208 - is #XTCPDATA correct there, or should it be #XTCPRECV?

File: applications/serial_lte_modem/doc/TCPIP_AT_commands.rst [hunk 1]
```diff
@@ -1572,7 +1572,7 @@ It is reported to the client as follows:
 
 ::
 
-   #XTCPCLI: <datatype>,<size>
+   #XUDPRECV: <datatype>,<size>
```
Comment (junqingzou) [inline applications/serial_lte_modem/doc/TCPIP_AT_commands.rst:1575]: No, for both TCP Server and TCP Client, the RX unsolicited is now:
  #XTCPDATA: <datatype>,<size>

Comment (ru-fu) [reply applications/serial_lte_modem/doc/TCPIP_AT_commands.rst:1575]: OK, that seems to be wrong in some of the examples - I can try and fix it.

This case here is for UDP though - should it be #XTCPDATA or #XUDPDATA?

Comment (junqingzou) [reply applications/serial_lte_modem/doc/TCPIP_AT_commands.rst:1575]: It's still "#XUDPRECV: < datatype >, < size >" unfortunately. Please change the doc to be "#XUDPDATA: < datatype >, < size >" and I'll update the code accordingly

Comment (ru-fu) [reply applications/serial_lte_modem/doc/TCPIP_AT_commands.rst]: Thanks - updated in the places I could find.

**Review by didrikr — APPROVED** [review]
Thanks @ru-fu. Looks good to me.

**Review by junqingzou — APPROVED** [review]
👍

**Review by rlubos — APPROVED** [review]

**Pull request merged**